configure

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env python
 
+import sys
+if sys.version_info >= (3, 0):
+  sys.stdout.write("Python 3.x is not compatible at this time\n")
```

Comment:
I'd also suggest the actual command to run, with a special case for macOS as it has no python2 symlink natively:

```python
if platform == "darwin":
  print('Python 3 is not supported. Please use: python2.7 configure')
else:
  print('Python 3 is not supported. Please use: python2 configure')
````